### ISSUE TITLE
ncm-network 18.6.0: why requiring a brodcast address when using a static bootproto

### ISSUE THREAD

**Issue by jouvin** [quattor/configuration-modules-core#1305]
With the new version of `ncm-network` (was previously running 17.7), `ncm-network` raises an error if there is no broadcast address defined for an interface using a static bootproto. If I'm right it used to be a warning and was working perfectly as the broadcast was computing by ifcfg from the network mask. Any good reason for this change?

**Comment by jrha** [comment]
Is the error flagged at compile or run time?

**Comment by jrha** [comment]
If it's only failed at run-time then I consider it a bug.

**Comment by jouvin** [comment]
Yes it happens only at run time, the compilation is just fine. I agree (and in fact wanted to mention it) but at the end, I think there is no reason to require a parameter which has a sensible default value set by the underlying configuration tools.

I didn't add it to #325 as I suspect the problem is there since `ncm-network` rewrite in 17.10 or 17.12... but if we agree that this change is not appropriate, it is probably easy to revert it to a warning before 18.6 release...

**Comment by jrha** [comment]
Lets give @stdweird a chance to respond.
